src/main/steps/c100-rebuild/safety-concerns/navigationController.test.ts

```diff
@@ -142,7 +142,7 @@ describe('SafteyConcernsNavigationController for only applicant', () => {
   });
 
   test('From children guidelines screen -> navigate to the children report abuse screen', async () => {
-    expect(C100Sequence[84].getNextStep({})).toBe(C100_C1A_SAFETY_CONCERNS_CONCERNS_ABOUT_CHILD);
+    expect(C100Sequence[87].getNextStep({})).toBe(C100_C1A_SAFETY_CONCERNS_CONCERNS_ABOUT_CHILD);
```

Comment:
Always avoid such changes, better add the your test cases at the end so that you don't need to change sequence for existing flows.

Reply:
I added it inside because it was connected to the children steps, next time should I just attach my part of the sequence to the end of file? @channa-patil

Reply:
should I move to end of the file?